yarn.lock

```diff
@@ -12278,7 +12278,7 @@ jest-watcher@^24.3.0, jest-watcher@^24.9.0:
     jest-util "^24.9.0"
     string-length "^2.0.0"
 
-jest-when@^2.7.2:
+jest-when@2.7.2:
```

Comment:
🤔 Seems that the `yarn.lock` validation in the CI doesn't work. The CI should fail if the lock file is not matched with the package.json.

Need further debug